<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>InputPage</class>
 <widget class="QWizardPage" name="InputPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>WizardPage</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="inputPathLabel">
     <property name="text">
      <string>Input path</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="inputPath">
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="addFileButton">
       <property name="text">
        <string>Add file</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="addDirectoryButton">
       <property name="text">
        <string>Add directory</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QCheckBox" name="batchMode">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>Batch mode</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="outputPathLabel">
     <property name="text">
      <string>Output path</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="outputPath">
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="outputPathButton">
     <property name="text">
      <string>Browse...</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
